tests/packagedcode/data/golang/gosum/sample1/go.sum

```diff
@@ -0,0 +1,4 @@
+github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
+github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
```

Comment:
How would you handle the dupes here, since they obviously about one and the same?
And also what to do about the checksums?

Reply:
@pombredanne 
> How would you handle the dupes here, since they obviously about one and the same?

Do we really need to handle dupes here? I think checksums are different so can we can add them differently.

>And also what to do about the checksums?

I was thinking that we should add checksums option in `DependentPackage` in `models.py` :)

Reply:
I am not that would work: 1. this is very specific to Go and 2. is the checksum really instead a resolved version for a Go module?

Reply:
Another way to think of this go.sums checksums could be to add a qualifier to the Purl... with key to be determined.

Reply:
If everything is looking good in this PR, we can see for checksums in another issue.

Reply:
OK, please enter a ticket for that part then.